dev/manual_tests/lib/tooltip_scroll_test.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This is still not the right place to test this. 
Framework unit tests in the flutter package are in the flutter package test directory: `packages/flutter/test/material/tooltip_test.dart`

Reply:
I see you have added some there, so it looks like this is not necessary.